generated/nidaqmx/system/device.py

```diff
@@ -130,6 +131,52 @@ def do_ports(self):
 
     # endregion
 
+    # region Calibration Info property
+
+    @property
+    def ext_cal_last_date_and_time(self):
+        """
+        Indicates the last date and time that the device underwent an
```

Comment:
FYI: Please consider enabling hyperlinks to the stdlib docs for datetime by enabling the intersphinx plugin. See https://github.com/ni/measurementlink-python/pull/430 for an example.

This is not required for this PR. I am posting this suggestion on multiple PRs.

Reply:
Noted, we could do it in another PR.